docs/quickstart.asciidoc

```diff
@@ -62,27 +60,36 @@ customresourcedefinition.apiextensions.k8s.io/kibanas.kibana.k8s.elastic.co crea
 customresourcedefinition.apiextensions.k8s.io/logstashes.logstash.k8s.elastic.co created
 ----
 
-. Install the operator with its RBAC rules:
+. Install the operator with its RBAC rules via link:https://kubernetes.io/docs/reference/kubectl/generated/kubectl_apply/[`apply`]:
```

Comment:
My recollection is that we should avoid latinisms (https://github.com/elastic/cloud-on-k8s/pull/5456):
```suggestion
. Install the operator with its RBAC rules with link:https://kubernetes.io/docs/reference/kubectl/generated/kubectl_apply/[`apply`]:
```